tensorrt_llm/_torch/models/modeling_deepseekv3.py

```diff
@@ -479,6 +497,38 @@ def __init__(self, model_config: ModelConfig[PretrainedConfig],
         if not self.deepseek_allreduce_disabled:
             self.deepseek_allreduce = DeepseekAllReduce(self.mapping)
 
+    def _compute_mlp_tp_size(self, intermediate_size: int,
+                             block_size: int) -> int:
+        """
+        In the case of Deepseek-R1, the TP size of MLP is capped by intermediate_size // block_size.
+        To avoid the costly inter-node all-reduce, we further restrict TP size to be
+        divisible by gpus_per_node.
```

Comment:
```suggestion
        a factor of gpus_per_node.
```

I made a mistake when modifying this part in PR 3572. Could you help correct it? Thanks.

Reply:
Done